### PR TITLE
feat: update templates

### PR DESCRIPTION
**Pull request by Sonichigo**
## What does this PR do?

This PR works towards making the PR templates and Issue Templates more friendly for contributors and maintainers to work with.

## Related PRs and Issues

N/A

## Type of change

- [x] Code style update (formatting, local variables)

## Test Plan

N/A

## Checklist:
- [x] Have you read the [Contributing Guidelines on issues](https://github.com/keploy/keploy/blob/master/CONTRIBUTING.md)?
- [x] My code follows the style guidelines of this project.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/--bug-report.yaml [hunk 1]
```diff
@@ -26,39 +26,49 @@ body:
     required: true
 - type: textarea
   attributes:
-    label: Steps to reproduce
-    description: Add steps to reproduce this behaviour, include console or network logs and screenshots
+    label: 👟 Reproduction steps
```
Comment (shivamsouravjha) [inline .github/ISSUE_TEMPLATE/--bug-report.yaml:29]: better name can be used

**Review by shivamsouravjha — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/--bug-report.yaml [hunk 2]
```diff
@@ -26,39 +26,49 @@ body:
...
+  validations:
+      required: true
 - type: dropdown
   id: version
   attributes:
-    label: Version
+    label: 🎲 Version
     options:
-      - Cloud
+      - Keploy Cloud
```
Comment (nehagup) [inline .github/ISSUE_TEMPLATE/--bug-report.yaml:63]: Remove this and ask for keploy version - textarea

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,45 +1,28 @@
...
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [ ] Code style update (formatting, local variables)
 - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
 - [ ] This change requires a documentation update
 
-## Please let us know if any test cases are added
+## Test Plan
 
 Please describe the tests(if any). Provide instructions how its affecting the coverage.
```
Comment (shivamsouravjha) [inline .github/PULL_REQUEST_TEMPLATE.md:21]: this can be more descriptive

File: .github/ISSUE_TEMPLATE/--bug-report.yaml [hunk 4]
```diff
@@ -55,14 +55,12 @@ body:
...
+- type: textarea
   id: version
   attributes:
     label: 🎲 Version
-    options:
-      - Keploy Cloud
-      - Self-hosted
-      - Local
+    value: |
+      "I am using latest version v2.0.0-alpha31"
```
Comment (nehagup) [inline .github/ISSUE_TEMPLATE/--bug-report.yaml:63]: Why is it not a placeholder.

**Pull request merged**